src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
```

Comment:
Avoid using tag names for styling elements directly, except for `html` and `body`. It's better to use classes to ensure that the styles are reusable and do not affect other elements with the same tag. Consider adding a class to the elements you want to style.